filter/ArticleCrossrefXmlFilter.php

```diff
@@ -172,6 +167,10 @@ public function createJournalArticleNode($doc, $submission)
                 $personNameNode->appendChild($node = $doc->createElementNS($deployment->getNamespace(), 'surname', htmlspecialchars(ucfirst($givenNames[$locale]), ENT_COMPAT, 'UTF-8')));
             }
 
+            if ($author->getData('orcid')) {
+                $personNameNode->appendChild($node = $doc->createElementNS($deployment->getNamespace(), 'ORCID', $author->getData('orcid')));
+            }
+
```

Comment:
This looks familiar to me and I'm not sure why. I think at some point there was some error that was caused by the relationship between orcid and given name. I _think_ this might have been it:

https://github.com/pkp/pkp-lib/issues/6852

Can you review that just to make sure that there isn't some specific reason why the `orcid` field was added above?

Reply:
(And if this change is reverted, maybe need to apply that fix to the OPS plugin, which puts `orcid` somewhere different.)

Reply:
Oh, yes, ORCID needs to go before alt-name. Good catch/memory!!! :+1: